jailbreaking/jailbreak-faq.md

```diff
@@ -9,50 +9,65 @@ nav_order: 98
 
 We recommend reading this article before or after jailbreaking your Kindle device and share it with other people.
 
+{: .note}
+Most of these instructions are subjected to changes due to newer jailbreaks, patches, extensions, firmware versions, etc.
```

Comment:
add a note of when this was written 

ex: accurate as of version 12.11.2, released oct 23, 2021